### DISCUSSION TITLE
PHP as supported language.

### DISCUSSION THREAD

**Discussion by jeffersoncechinel** [sqlc-dev/sqlc#2047] [Feature Requests & Ideas]
I don't see any plans for the PHP language support in SQLC website. Was it already discussed? Any thoughts? I think the whole PHP community would benefit from it. Me personally would love to leave behind those monsters ORMs and ActiveRecords in favor of this lighter and query controlled approach. This project is awesome! Thank you.

**Comment by kyleconroy** [comment]
Python and Kotlin support work through plugins, so someone (maybe you?) could write a PHP plugin if you wanted.

Now that PHP supports WASM, you could even do it in PHP too!
